src/column-layout/interfaces.ts

```diff
@@ -27,6 +31,14 @@ export interface ColumnLayoutProps extends BaseComponentProps {
    */
   disableGutters?: boolean;
 
+  /**
+   * Use together with `columns` to specify the desired minimum width for each column in pixels.
+   *
+   * The number of columns is determined based on the value of this property, the available space,
```

Comment:
Suggestion: "The number of columns is determined by the value of  this property, the available space, ...."

Reply:
done!